### PR TITLE
Update pr-welcome-msg with packit tests info

### PR DESCRIPTION
**Pull request by fernflower**
Let's mention the big leap for leapp officially together with a command to (re-)trigger tests.

### CODE REVIEW

File: .github/workflows/pr-welcome-msg.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ jobs:
             - **review please** to notify leapp developers of review request
```
Comment (pirat89) [inline .github/workflows/pr-welcome-msg.yml:24]:
```suggestion
            - **review please <at>oamg/developers ** to notify leapp developers of review request if nobody answered for couple of days
```

File: .github/workflows/pr-welcome-msg.yml [hunk 2]
```diff
@@ -24,6 +24,8 @@ jobs:
...
             - **/packit copr-build** to submit a public copr build using packit
 
+            Packit will automatically schedule regression tests for this PR's build and leapp\*master\*. If you need a different version of leapp from PR#42, use `/packit test oamg/leapp#42`
```
Comment (pirat89) [inline .github/workflows/pr-welcome-msg.yml:27]:
```suggestion
            Packit will automatically schedule regression tests for this PR's build and latest upstream (devel) build of leapp. If you need a different version of leapp from PR#42, use `/packit test oamg/leapp#42`
```

@fernflower   I prefer to rather skip the name of the master/main/upstream branch as we are anyway enforced to change it soon to something else.

Comment (fernflower) [reply .github/workflows/pr-welcome-msg.yml:27]: Ok, I will use `latest upstream build` instead, thanks

File: .github/workflows/pr-welcome-msg.yml (hunk 1 above)
Comment (fernflower) [reply .github/workflows/pr-welcome-msg.yml]: Done

File: .github/workflows/pr-welcome-msg.yml (hunk 2 above)
Comment (fernflower) [reply .github/workflows/pr-welcome-msg.yml]: Done everywhere

**Comment by fernflower** [comment]
Hmmm I am pretty sure there has been no reaction to leapp-ci build command, so I will add another commit to remove the note as well. I guess we have moved on for good :)

File: .github/workflows/pr-welcome-msg.yml [hunk 3]
```diff
@@ -21,15 +21,17 @@ jobs:
             ## **Thank you for contributing to the Leapp project!**
             Please note that every PR needs to comply with the [Leapp Guidelines](https://leapp.readthedocs.io/en/latest/contributing.html#) and must pass all tests in order to be mergeable.
             If you want to request a review or rebuild a package in copr, you can use following commands as a comment:
-            - **review please** to notify leapp developers of review request
+            - **review please @oamg/developers** to notify leapp developers of the review request
```
Comment (pirat89) [inline .github/workflows/pr-welcome-msg.yml:24]: @fernflower  I tried to replace the `@` by `<at>` to be sure that people are not pinged with each new welcome msg. Would this lead to such a situation?

Comment (fernflower) [reply .github/workflows/pr-welcome-msg.yml:24]: From what I see in a test run https://github.com/fernflower/leapp-repository/pull/17#issuecomment-1521570808 that's not the case. I'd leave a `@` here and submit a patch later if we are not happy.

Comment (pirat89) [reply .github/workflows/pr-welcome-msg.yml:24]: @fernflower thanks. you are right, it seems ok. let's merge!

**Review by pirat89 — APPROVED** [review]

**Pull request merged**